### PR TITLE
Improve create-new-object docs with example

### PR DESCRIPTION
**Pull request by davideghz**
https://github.com/yourlabs/django-autocomplete-light/issues/918#issuecomment-333366515

### CODE REVIEW

**Comment by davideghz** [comment]
umh, why does it break tests?

File: docs/tutorial.rst [hunk 1]
```diff
@@ -349,7 +349,13 @@ the view which will do ``Country.objects.create(name='Tibet')``. It will be
 included in the server response so that the script can add it to the widget.
 
 Note that creating objects is only allowed to staff users with add permission
```
Comment (jpic) [inline docs/tutorial.rst]: Should we change this so that we don't need to be staff user anymore ?

**Comment by jpic** [comment]
Don't worry about it, some changes occured upstreams I'll fix them when adding dj2.0 ;)

Do we really still need to be staff user ?

File: docs/tutorial.rst (hunk 1 above)
Comment (davideghz) [reply docs/tutorial.rst]: The point is. Staff members get this privilege by default; to grant same right to other users (not staff members) it's necessary to set the `add` permission explicitly.

**Comment by jpic** [comment]
I'm really not sure, I don't see any requirement to have a user with is_staff=True in the source code, are you sure this is necessary ?

**Comment by davideghz** [comment]
I see what you mean. As per my tests, a staff user was able to create new objects, while a standard user wasn't. As per Django doc, however, being a staff user does not implies having create rights on resources. I will try it out on a new test project and follow up!

**Comment by jpic** [comment]
I've just tested in the test project and I can confirm that is_staff is not necessary anymore, only being logged in and having create permission is required.

Could you update the docs to match this please ?
Thanks a HEAP for your effort, rock on :guitar: :saxophone: :trumpet:

**Comment by davideghz** [comment]
Cool! Thx for testing!

**Comment by jpic** [comment]
Just perfect, your example really helped me out !
But at the end of the day I was using Permssion.objects.get(name='add_mymodel') which was a bit easier than filtering by name, did you try that yourself ?

**Comment by davideghz** [comment]
well I use it in a `post_save` signal exactly as I wrote and it works perfectly :D
My model is called `DjProfile` and I use it like:
```
@receiver(post_save, sender=User)
def grant_permissions(sender, instance, created, **kwargs):
    if created:
        permission = Permission.objects.get(name='Can add dj profile')
        instance.user_permissions.add(permission)
```
And yeah, maybe using groups would even be a better approach than setting the permission user by user.. I will think about it :)

**Comment by jpic** [comment]
Nice, thanks a heap for your contribution, be sure that it will help the community ;)

Best <3

**Pull request merged**